addons/Quiz/src/presenter.js

```diff
@@ -722,6 +844,40 @@ function AddonQuiz_create() {
         return model;
     };
 
+    presenter.addLangTag = function AddonTable_upgradeLangTag(model) {
+        var upgradedModel = {};
+        $.extend(true, upgradedModel, model);
+
+        if (!model['langAttribute']) {
```

Comment:
Tutaj raczej nie powinno to powodować problemów, ale na przyszłość trzeba z takim sprawdzaniem ostrożnie. Przykładowo, gdyby wartość propertiesa była ustawiona na string "0", to to będzie rzutowane na false i spełni taki warunek. Lepiej jest po prostu przyrównywać do undefined.

Reply:
Zamieniłem aby sprawdzało czy jest undefined